Command/CreateClientCommand.php

```diff
@@ -67,13 +68,27 @@ protected function configure(): void
                 InputArgument::OPTIONAL,
                 'The client secret'
             )
+            ->addOption(
+                'public',
+                null,
+                InputOption::VALUE_NONE,
+                'Create a public client'
```

Comment:
@X-Coder264 OMG, just noticed there's no dot at the end here while all the other options have one. Cannot... unsee... ocd... to strong...

Reply:
😂

Reply:
But all arguments have no dot at the end, what a major inconsistency 😱

Reply:
I know, this is killing me now...

Reply:
🤣 😂

Reply:
What is killing me is this unresolved discussion... 😂